tests/singletest.php

```diff
@@ -17,4 +17,5 @@
 $manager->removeLogAndBackupFiles($logfilename);
 //$manager->setTestDirectory(getcwd());
 $manager->setDatabase($db);
-$manager->runFileTest(new TextReporter(), $argv[1]);
```

Comment:
what was the warning?  It didn't like that it was an anonymous (no-handle) object?